scripts/yggdrasil-iptunnel/yggdrasil-setup

```diff
@@ -4,36 +4,137 @@
 echo 1 > /proc/sys/net/ipv4/ip_forward
 echo 1 > /proc/sys/net/ipv6/conf/all/forwarding
 
-SUBNET4="10.10.0."
+function confset {
+    section="$1"
+    key="$2"
+    value="$3"
+    file="$4"
+
+    # If file is missing create one
+    if [[ ! -f "$file" ]]; then 
+	    touch "$file"
+    fi
+
+    # If file is section create it
+    if [[ -z "$(grep "\[$section\]" "$file")" ]]; then
+        echo "[$section]" >> "$file"
+    fi
+
+    # If file is missnig key add it
+    confget -f "$file" -s "$section" -c "$key"
+    res=$?
+    if [[ "$res" == "1" ]]; then
+        sed -i "s/\[$section]/\[$section]\n$key=$value/" "$file"
+    else
+    # Otherwise change it
+        sed -i "/^\[$section]/,/^\[/{s/^$key=.*/$key=$value/;}" "$file"
+    fi
+}
 
 # Give yggdrasil enough time to create an ygg0 interface before we start adding routes
-sleep 5
+sleep 3
+if [ -f /etc/yggdrasil.iptunnel.conf ]; then 
+   touch /etc/yggdrasil.iptunnel.conf
+fi
+IPv6NAT=$(confget -f "/etc/yggdrasil.iptunnel.conf" -s "iptunnel" IPv6nat)
```

Comment:
Why are all these variables missing underscores between words?

Reply:
Because they reflect the doc, prob will get cleaned up in next sweep anyway.

Reply:
I would do it before the merge if you can, not a big deal though.

Reply:
I was thinking of re-wroking the config file a bit anyway in the next iteration
i just dont want to break anythign right now